### ISSUE TITLE
Unimplemented pyOCD function means nrfjprog unusable

### ISSUE THREAD

**Issue by szmoore** [mjdietzx/nrfjprog#19]
In Debian 8.4 GNU/Linux with requirements installed as per instructions.
Run `nrfjprog ids`
Output:
Displaying the serial numbers of all debuggers connected to the PC.
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/nrf51sdk-10.0.0/nrfjprog/nrfjprog/__main__.py", line 313, in <module>
    main()
  File "/opt/nrf51sdk-10.0.0/nrfjprog/nrfjprog/**main**.py", line 309, in main
    cli.run()
  File "/opt/nrf51sdk-10.0.0/nrfjprog/nrfjprog/**main**.py", line 82, in run
    self.args.func(self.args)
  File "/opt/nrf51sdk-10.0.0/nrfjprog/nrfjprog/model/perform_command.py", line 79, in ids
    perform_command_jlink.ids(args) if is_jlink() else perform_command_daplink.ids(args)
  File "/opt/nrf51sdk-10.0.0/nrfjprog/nrfjprog/model/perform_command_daplink.py", line 66, in ids
    assert(False), 'Not implemented in pyOCD.' # TODO: Implement this in pyOCD.
AssertionError: Not implemented in pyOCD.

**Comment by mjdietzx** [comment]
Hey,

I just implemented the ids command to work with pyOCD so it will work with this latest commit: 0e90ec95431b4a3f98e31dd264db9fb20aa81262.

As I just added pyOCD support there are still a few commands that aren't implemented and will let you know with the AssertionError and message. For now you will not be able to use these commands (however the common ones all work). pyOCD should be 100% supported in a day or two.
